pipeline_steps/deploy.groovy

```diff
@@ -55,6 +68,13 @@ def upgrade(String stage_name, String upgrade_script, List env_vars) {
           sh "git reset --hard"
         }
         common.prepareRpcGit()
+        if ( stage_name == "Leapfrog Upgrade" ) {
+          create_test_resource_load(
+            env.LEAPFROG_TEST_SERVERS as Integer,
```

Comment:
Naming is hard I know, but this load thing could be used for intra-series too, so it's good to rename this env variables.

GENERATE_TEST_{SERVERS,NETWORKS,VOLUMES}